test/test-helpers.js

```diff
@@ -112,12 +112,21 @@ let fakeEnvironment = {
     this.clock.restore();
     videojs.xhr.XMLHttpRequest = window.XMLHttpRequest;
     this.xhr.restore();
+    ['warn', 'error'].forEach((level) => {
+      if (videojs.log && videojs.log[level] && videojs.log[level].restore) {
+        videojs.log[level].restore();
+      }
+    });
+
   }
 };
 
 export const useFakeEnvironment = function() {
   fakeEnvironment.clock = sinon.useFakeTimers();
-
+  ['warn', 'error'].forEach((level) => {
+    sinon.stub(videojs.log, level);
```

Comment:
I agree that the warnings are a pain in the tests, but are they something we should address? Or can we never get around them, as we need to use the internals of videojs?